homeassistant/components/myuplink/sensor.py

```diff
@@ -188,8 +198,27 @@ def native_value(self) -> str:
         device_point = self.coordinator.data.points[self.device_id][self.point_id]
         return self.options_map[str(int(device_point.value))].capitalize()  # type: ignore[no-any-return]
 
-    @property
-    def extra_state_attributes(self) -> dict[str, str]:
-        """Add extra state attributes."""
-        device_point = self.coordinator.data.points[self.device_id][self.point_id]
-        return {"raw_value": str(int(device_point.value))}
+
+class MyUplinkEnumRawSensor(MyUplinkDevicePointSensor):
+    """Representation of a myUplink device point sensor for raw value from ENUM device_class."""
+
+    def __init__(
+        self,
+        coordinator: MyUplinkDataCoordinator,
+        device_id: str,
+        device_point: DevicePoint,
+        entity_description: SensorEntityDescription | None,
+        unique_id_suffix: str,
+    ) -> None:
+        """Initialize the sensor."""
+        super().__init__(
+            coordinator=coordinator,
+            device_id=device_id,
+            device_point=device_point,
+            entity_description=entity_description,
+            unique_id_suffix=unique_id_suffix,
+        )
+
+        self._attr_device_class = None
+        self._attr_entity_registry_enabled_default = False
+        self._attr_name = f"{self.name} raw"
```

Comment:
I'm not sure that this will work. This is safer.
```suggestion
        self._attr_name = f"{device_point.parameter_name} raw"
```

Reply:
OK